src/app/content/highlights/components/Highlights.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
```

Comment:
breaking things into smaller files is cool, but you're gonna have hella conflicts with karina's changes

Reply:
I assume I'll have to do this anyway so let me know which way you see better for project management

Reply:
you can continue as you are, just be very aware of karina's changes to the old component files